monai/transforms/post/array.py

```diff
@@ -83,6 +84,9 @@ def __call__(
             ValueError: When ``self.other=None`` and ``other=None``. Incompatible values.
 
         """
+        if isinstance(img, (tuple, list)):
+            return [self(img=i.unsqueeze(0), sigmoid=sigmoid, softmax=softmax, other=other).squeeze(0) for i in img]
```

Comment:
This looks good. Can we think of any way to abstract this out to a base level to reduce repetition when we implement it in other post-processing transforms?

Reply:
If we can think of a way to abstract this feature to base class, it would be better, but I didn't get so far...

Thanks.